src/database/schema/ChannelListMemberSchema.ts

```diff
@@ -86,6 +86,8 @@ class ChannelListMemberSchema implements BaseDbSchema {
     }
   };
 
+  //! TODO:
+  //! UPDATE ANONYMOUSID FOR SIGNED CHAT
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The comment indicates that there is a need to update the anonymousId for signed chat. However, no code changes have been made in this hunk. It's important to ensure that the necessary updates are made or remove the comment if it's not applicable.
```diff
-  //! TODO:
-  //! UPDATE ANONYMOUSID FOR SIGNED CHAT
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->